tsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
   "compilerOptions": {
+    "baseUrl": "./",
```

Comment:
Let's remove this for now, it breaks typechecking for styled-components as there is a matching declaration in this dir. Happy to add import shortcuts in a different PR!